tasmota/tasmota_xdrv_driver/xdrv_05_irremote_full.ino

```diff
@@ -221,37 +222,75 @@ void IrReceiveInit(void)
   irrecv->enableIRIn();                  // Start the receiver
 }
 
+namespace {
```

Comment:
Just curious, why a namespace?

Reply:
Honestly, just a habit.
I mean, they _technically_ are meant to be somewhat better (see [here](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf22-use-an-unnamed-anonymous-namespace-for-all-internalnon-exported-entities),  [here](https://stackoverflow.com/questions/4422507/superiority-of-unnamed-namespace-over-static) or [here](https://abseil.io/tips/186) for rationale), but this doesn't fully apply to here and I have no strong preference either way....
Would you like them removed?

Reply:
Ah ok. This part of C++ is obscure to me. Here it's a better version of `static`. Leave it as it is.